plugins/unicodelib/unicodelib.cpp

```diff
@@ -286,28 +292,43 @@ class UPCList // User perceived character list
     uint32_t  length_;
     uint32_t  capacity_;
     bool invalid_;
-    void doCreateUPCList(BreakIterator& cbi) {
+    void doCreateUPCList(BreakIterator* cbi) {
```

Comment:
minor: Better to keep this as a reference parameter and pass *cnbi in.  (It clarifies this function does not support NULL).